app/src/main/java/by/evgen/an11pazniakhomework14/homework15/util/Util.kt

```diff
@@ -0,0 +1,50 @@
+package by.evgen.an11pazniakhomework14.homework15.util
+
+import by.evgen.an11pazniakhomework14.homework15.data.*
+import by.evgen.an11pazniakhomework14.old.CatDataClass
+import by.evgen.an11pazniakhomework14.old.InheritCatNoDataClass
+
+fun WhiskeyCola.getInfo() =
+    "name: " + this.name + ", capacity: " + this.volume + ", alcohol: " + this.alco
```

Comment:
you can use "name:  ${this.name}, capacity: ${this.volume }, alcohol:  ${this.alco}"